static/bidder-info/pgam.yaml

```diff
@@ -0,0 +1,16 @@
+maintainer:
+  email: "ppatel@pgammedia.com"
+capabilities:
```

Comment:
You don't support traffic subject to GDPR?

Reply:
Is it related to yaml ? the host company has gdpr support, but janet has no vendor id

Reply:
Yes, this is where you would put your vendor ID in the yaml. I wanted to ensure that this was not simply overlooked.